client/package.json

```diff
@@ -2,7 +2,7 @@
     "name": "superdesk",
     "license": "GPL-3.0",
     "scripts": {
-        "postinstall": "cd extensions/belgaCoverage && npm install && npm run compile"
+        "postinstall": "node ./tasks/install-extensions.js"
```

Comment:
You shouldn't need to change the setup neither add any additional scripts. Is something not working as is?

Reply:
The `postinstall` script install and compile only the `belgacoverage extension`. Shouldn't it be modified to install and compile all the extensions available inside the extension folder?

Reply:
Sorry, my bad, you do need to compile the new extension. I would have simply added another `cd /extension-path && npm run compile` command to `postinstall`, but a script also works.